source/localizable/components/defining-a-component.md

```diff
@@ -41,37 +68,73 @@ export default Ember.Route.extend({
   }
 });
 ```
+Note that the properties passed to the component stay in sync (technically known as being "bound"). That is, if the value of componentProperty changes in the component, outerProperty will be updated to reflect that change. The reverse is true as well.
```

Comment:
Even if it's an anti-pattern, I think it's worth highlighting so that developers are aware of potential side-effects.